src/Mvc/test/WebSites/TagHelpersWebSite/Views/Home/GlobbingTagHelpers.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
+    ViewData["Title"] = "Globbing Page";
+}
+
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+@section footerContent {
+    <!-- File wildcard -->
+    <script asp-src-include="/js/dist/dashboard*.js"></script>
+    <!-- RazorClassLib folder wildcard -->
+    <script asp-src-include="/_content/RazorPagesClassLibrary/**/file.js"></script>
```

Comment:
@javiercn this is the type of test you were concerned about not working right? 

Also with the suggestions to plumb through the `IFileProvider` by using `DirectoryInfoBase`s; what's your concern there? Asking because it seems like a significant, risky piece of work for not a lot of gain.